### PR TITLE
Add Mission Parameter Information to ace wiki

### PR DESCRIPTION
**Pull request by PabstMirror**
Documentation for PR #2826

### CODE REVIEW

File: wiki/framework/settings-framework.md [hunk 1]
```diff
@@ -49,9 +50,10 @@ The load order for the settings are:
 1. Mod Config
 2. Server Config
 3. Mission Config
-4. Placed Mission Modules
+4. Mission Paramaters
+5. Placed Mission Modules
 
-What this means is that at any the 3 points after the Mod Config it is possible to insert your adjusted settings and force those (optionally). This is a powerful tool for server admins, whom can ensure that everyone is using uniform settings across the board on their server. And it provides mission makers the ability to easily set settings for their mission, without creating a large dependency on ACE3; you do not have to place down mission modules.
+What this means is that at any the 4 points after the Mod Config it is possible to insert your adjusted settings and force those (optionally). This is a powerful tool for server admins, whom can ensure that everyone is using uniform settings across the board on their server. And it provides mission makers the ability to easily set settings for their mission, without creating a large dependency on ACE3; you do not have to place down mission modules.
```
Comment (bux578) [inline wiki/framework/settings-framework.md]: `whom can ensure`? Not sure, sounds wrong to me.

File: wiki/framework/settings-framework.md [hunk 2]
```diff
@@ -103,3 +105,40 @@ class ACE_Settings {
 As stated before, the server config gets loaded through the optional `ace_server.pbo`. This PBO is only required (and should only be used) on the server - clients do not need to have this! It is for this reason we have not signed this PBO.
 
 Load the `ace_server.pbo` like any other addon on your server. It is advised to create an `@aceServer` mod folder with an `addons` sub folder where you would paste the `ace_server.pbo` and load that through `-serverMod=@aceServer`.
+
+
+### 3.3 Changing ACE_settings dynamically with Mission Parameters
+
+It is possible to change ace settings via a mission's params.
```
Comment (bux578) [inline wiki/framework/settings-framework.md]: `ACE3 settings` via [Mission Parameters](https://community.bistudio.com/wiki/Arma_3_Mission_Parameters)
maybe add a link to: https://community.bistudio.com/wiki/Arma_3_Mission_Parameters

File: wiki/framework/settings-framework.md [hunk 3]
```diff
@@ -103,3 +105,40 @@ class ACE_Settings {
...
+This can allow a single mission to be played with different realism settings.
+They are read after all other configs but before modules. Params are forced so they will override any module setting.
+
+#### Basic Setup:
+1. Add a param with the same name as an ace_setting
```
Comment (bux578) [inline wiki/framework/settings-framework.md]: Mission Parameter

File: wiki/framework/settings-framework.md [hunk 4]
```diff
@@ -103,3 +105,40 @@ class ACE_Settings {
 As stated before, the server config gets loaded through the optional `ace_server.pbo`. This PBO is only required (and should only be used) on the server - clients do not need to have this! It is for this reason we have not signed this PBO.
 
 Load the `ace_server.pbo` like any other addon on your server. It is advised to create an `@aceServer` mod folder with an `addons` sub folder where you would paste the `ace_server.pbo` and load that through `-serverMod=@aceServer`.
+
+
+### 3.3 Changing ACE_settings dynamically with Mission Parameters
+
+It is possible to change ace settings via a mission's params.
+This can allow a single mission to be played with different realism settings.
+They are read after all other configs but before modules. Params are forced so they will override any module setting.
```
Comment (bux578) [inline wiki/framework/settings-framework.md]:
```
- Params
+ Mission Parameters
```

File: wiki/framework/settings-framework.md [hunk 5]
```diff
@@ -103,3 +105,40 @@ class ACE_Settings {
...
+
+#### Basic Setup:
+1. Add a param with the same name as an ace_setting
+2. Add the `ACE_setting = 1;` flag
+3. Add `values[]` and optionally `texts[]` to match the setting
+
+Note: For now only BOOL or SCALAR settings are supported.
```
Comment (bux578) [inline wiki/framework/settings-framework.md]:
```
<div class="panel callout">
    <h5>Please note:</h5>
    <p>For now only <code>BOOL</code> or <code>SCALAR</code> settings are supported..</p>
</div>
```

File: wiki/framework/settings-framework.md [hunk 6]
```diff
@@ -103,3 +105,40 @@ class ACE_Settings {
...
+        title = "$STR_ACE_Repair_addSpareParts_name"; //You can use ACE's stringtables
+        ACE_setting = 1;
+        values[] = {0, 1}; //setting is a BOOL, but values still need to be numbers, so 0 is false, 1 is true
+        texts[] =  {"False", "True"};
+        default = 1;
+    };
+};
+```
+
+Review the [biki](https://community.bistudio.com/wiki/Arma_3_Mission_Parameters) for more information on setting up parameters.
```
Comment (bux578) [inline wiki/framework/settings-framework.md]: Mission Parameters

File: wiki/framework/settings-framework.md (hunk 1 above)
Comment (MikeMatrix) [inline wiki/framework/settings-framework.md]: Whom is incorrect here. Simple rule of thumb is to check if you can replace whom with him/her. It's correct if that's possible, but incorrect when he/she fits better.
Example here being
`He can ensure that everyone is using uniform settings [...]`
which is who not whom.

**Pull request merged**